examples/dev-mashup/README.md

```diff
@@ -0,0 +1,27 @@
+# Nebula Development Mashup example
+
+Example mashup for nebula development
+
+## Install
+
+Update dependencies
+
+```
+yarn
+```
+
+To use a local nebula version, run this in the stardust folder to link you local version.
+
+```
+yarn link @nebula.js/stardust
```

Comment:
unsure about this line, so needs feedback.

Reply:
yarn link "@nebula.js/stardust" should be run in the folder where we want to have stardust. So in this case we should run it in local-mashup folder.